src/CBlueprint.h

```diff
@@ -131,42 +132,42 @@ extern "C" {
     SC_API void sc_blueprint_free(sc_blueprint_t* blueprint);
 
     /** \returns Blueprint name */
-    SC_API const char* sc_blueprint_name(const sc_blueprint_t* blueprint);
+    SC_API const char* sc_blueprint_name(const sc_blueprint_t* handle);
 
     /** \returns Blueprint description */
-    SC_API const char* sc_blueprint_description(const sc_blueprint_t* blueprint);
+    SC_API const char* sc_blueprint_description(const sc_blueprint_t* handle);
 
     /*----------------------------------------------------------------------*/
 
-    /** \returns Metadata array handle */
-    SC_API const sc_metadata_collection_t* sc_metadata_collection_handle(const sc_blueprint_t* blueprint);
+    /** \returns Metadata Collection array handle */
+    SC_API const sc_metadata_collection_t* sc_metadata_collection_handle(const sc_blueprint_t* handle);
 
-    /** \returns size of Metadata array */
-    SC_API size_t sc_metadata_collection_size(const sc_metadata_collection_t* metadata);
+    /** \returns size of Metadata Collection array */
+    SC_API size_t sc_metadata_collection_size(const sc_metadata_collection_t* handle);
 
     /*----------------------------------------------------------------------*/
 
     /** \returns Metadata at `index` handle */
-    SC_API const sc_metadata_t* sc_metadata_handle(const sc_metadata_collection_t* metadata_col, size_t index);
+    SC_API const sc_metadata_t* sc_metadata_handle(const sc_metadata_collection_t* handle, size_t index);
 
     /** \returns Metadata key */
-    SC_API const char* sc_metadata_key(const sc_metadata_t* metadata);
+    SC_API const char* sc_metadata_key(const sc_metadata_t* handle);
 
     /** \returns Metadata value */
-    SC_API const char* sc_metadata_value(const sc_metadata_t* metadata);
+    SC_API const char* sc_metadata_value(const sc_metadata_t* handle);
 
     /*----------------------------------------------------------------------*/
 
     /** \returns Resource Group Collection array handle */
-    SC_API const sc_resource_group_collection_t* sc_resource_group_collection_handle(const sc_blueprint_t* blueprint);
+    SC_API const sc_resource_group_collection_t* sc_resource_group_collection_handle(const sc_blueprint_t* handle);
 
     /** \returns size of Resource Group Collection array */
-    SC_API size_t sc_resource_group_collection_size(const sc_resource_group_collection_t* resource);
+    SC_API size_t sc_resource_group_collection_size(const sc_resource_group_collection_t* handle);
 
     /*----------------------------------------------------------------------*/
 
-    /** \returns Resource Group handle */
-    SC_API const sc_resource_group_t* sc_resource_group_handle(const sc_resource_group_collection_t* resource, size_t index);
+    /** \returns Resource Group at `index` handle */
```

Comment:
I would say this comment is not much of an improvement. It should go probably with something like "Resource Group handle at "index" in the array of ... "